adagrams/game.py

```diff
@@ -1,85 +1,113 @@
 from random import randint
-from adagrams.asset import LETTER_POOL
-
+# from adagrams.asset import LETTER_POOL
+
+LETTER_POOL = {
```

Comment:
It would have been fine to continue to use the copy of LETTER_POOL from your asset file. It's structure was fine.